cg/services/run_devices/abstract_classes.py

```diff
@@ -83,6 +87,27 @@ def post_process(self, run_name: str, dry_run: bool = False):
         """Store sequencing metrics in StatusDB and relevant files in Housekeeper."""
         pass
 
+    def post_process_all(self, run_names: list[str], dry_run: bool = False) -> bool:
```

Comment:
I forgot what we decided. Did we want to keep the post_process_all now ?

Reply:
yes, what we agreed was to improve the error handling and reduce the boolean juggling

Reply:
started new branch with this option, resolving